### DISCUSSION TITLE
Cannot find Python console

### DISCUSSION THREAD

**Discussion by chenjinhu** [Vector35/binaryninja-api#6286] [Q&A] [answered]
i connot find Python console.
I'm trying to learn how to develop plugins, but I can't find a place where I can execute Python code.
![image](https://github.com/user-attachments/assets/39ad283b-7d84-484e-a035-f15ea27f64a9)

connect_vscode_debugger(port=12345)

**Comment by psifertex** [comment, marked as answer]
What version are you using? Only paid editions have the Python console. If it's a paid version, do you see any errors in the error log when it launches?
